README.md

```diff
@@ -8,7 +8,7 @@ The JavaScript SDK Managed Download can be used for custom S3 downloads. The cli
 
 This package only works in Node.js versions 6+ currently. Use the npm package manager for Node.js to install the Managed Download package. Type the following command in a terminal window.
 
```

Comment:
The docs [here](https://awslabs.github.io/s3-managed-download-js/) need to be updated to  replace `@aws/s3-managed-download` with `@aws/s3-managed-downloader` in several places.